iphone/Classes/MediaModule.m

```diff
@@ -2095,8 +2095,14 @@ - (void)imagePickerController:(UIImagePickerController *)picker_ didFinishPickin
     if (cropRect != nil) {
         [dictionary setObject:cropRect forKey:@"cropRect"];
     }
-	
+    
     [self sendPickerSuccess:dictionary];
+    
+#ifndef TI_USE_KROLL_THREAD
+    //TIMOB-24389 : iOS: Appcelerator Titanium App Crashes after some usage time
+    KrollContext *krollContext = [self.pageContext krollContext];
+    [krollContext forceGarbageCollectNow];
+#endif
```

Comment:
Can we move this to `dispatchCallback:` instead, so it's ensured to be called for video capturing & editing as well? Or do you see down-sides there?